egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -165,7 +165,7 @@ def __init__(
         grad_scale_factor: float = 0.9,
```

Comment:
If we are applying this on batches of 10 frames, then probably this default for grad_scale_factor is too large, should be e.g. 0.5.